vendor/Makefile

```diff
@@ -6,16 +6,14 @@ BUILD_LIST_TARGET ?= list/all
 deps:
 	apk add alpine-sdk shadow
 
-## List all packages which have been updated relative to `master`
+## List only updated packages or all if apk, tasks or codefresh folders was updated
 list/updated:
-	@git --no-pager diff --name-only $(CURRENT_BRANCH) `git merge-base $(CURRENT_BRANCH) origin/$(TARGET_BRANCH)` | \
-		xargs -n 1 -r dirname | \
-		sort -u | \
-		grep -E '^vendor/' | \
-		sed 's,vendor/,,' | \
-		cut -d/ -f1 | \
-		sort -u | \
-		grep -E '.+'
+	$(eval CHANGES=$(shell git --no-pager diff --name-only origin/$(TARGET_BRANCH) | \
```

Comment:
Add `#` comments that describe in plain english what this does

Reply:
To workflows:

1) if `apk|tasks|codefresh` changes, then rebuild all
2) if `vendor` changes, then just rebuild those changed/updated packages